### PR TITLE
doc: update the external doc format

### PR DESCRIPTION
**Pull request by parth-gr**
Cephobjectstore is not formated correctly
https://rook.io/docs/rook/v1.12/CRDs/Cluster/external-cluster/#connect-to-an-external-object-store

**Description of your changes:**

**Which issue is resolved by this Pull Request:**
Resolves #

**Checklist:**

- [ ] **Commit Message Formatting**: Commit titles and messages follow guidelines in the [developer guide](https://rook.io/docs/rook/latest/Contributing/development-flow/#commit-structure).
- [ ] **Skip Tests for Docs**: If this is only a documentation change, add the label `skip-ci` on the PR.
- [ ] Reviewed the developer guide on [Submitting a Pull Request](https://rook.io/docs/rook/latest/Contributing/development-flow/#submitting-a-pull-request)
- [ ] [Pending release notes](https://github.com/rook/rook/blob/master/PendingReleaseNotes.md) updated with breaking and/or notable changes for the next minor release.
- [ ] Documentation has been updated, if necessary.
- [ ] Unit tests have been added, if necessary.
- [ ] Integration tests have been added, if necessary.

### CODE REVIEW

**Review by BlaineEXE — DISMISSED** [review]

**Comment by parth-gr** [comment]
@galexrt

File: Documentation/CRDs/Cluster/external-cluster.md [hunk 1]
```diff
@@ -180,9 +180,13 @@ If not installing with Helm, here are the steps to install with manifests.
 ### Connect to an External Object Store
 
 Create the object store resources:
+
 1. Create the [external object store CR](https://github.com/rook/rook/blob/master/deploy/examples/object-external.yaml) to configure connection to external gateways.
+
 2. Create an [Object store user](https://github.com/rook/rook/blob/master/deploy/examples/object-user.yaml) for credentials to access the S3 endpoint.
```
[the remark this answers is not in the document]
Comment (galexrt) [reply Documentation/CRDs/Cluster/external-cluster.md:185]: @parth-gr do you want to wait for a response or should we get the PR merged already?

[the remark this answers is not in the document]
Comment (parth-gr) [reply Documentation/CRDs/Cluster/external-cluster.md:185]: Not needed with this PR,
It seems okay
if required I will open a different PR to solve that.

**Pull request merged**